### PR TITLE
Prevent removing leading zero of postcode

### LINKED ISSUE #318: Les codes postaux 0XXXXX ne sont pas trouvés
Addok retire les premiers zéros des codes postaux et ne les trouve plus dans la recherche : 

> http://ban.addok.xyz/search?q=37 RUE DE CROIX 02230 FRESNOY LE GRAND
> Résultat : Rue de Fresnoy 02110 Croix-Fonsomme

Le résultat est correct sur cette version : 

> http://api-adresse.data.gouv.fr/search/?q=37 RUE DE CROIX 02230 FRESNOY LE GRAND
> Résultat : 37 Rue de Croix 02230 Fresnoy-le-Grand

Discussion initiale : https://forum.etalab.gouv.fr/t/comprendre-la-raison-dun-resultat-avec-explain-addok/3513/5

### PR DESCRIPTION
**Pull request by yohanboniface**
fix addok/addok#318

### CODE REVIEW

File: addok_france/utils.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def fold_ordinal(s):
 
 def remove_leading_zeros(s):
     """0003 => 3."""
```
Comment (davidbgk) [inline addok_france/utils.py:128]: Update the comment to reflect post codes exclusion?

**Review by davidbgk — APPROVED** [review]

File: addok_france/utils.py (hunk 1 above)
Comment (yohanboniface) [reply addok_france/utils.py:128]: Humm, I'd say the spirit of the function hasn't changed, so I'd keep the docstring focused on it. And let unittests monitor implementation details.
Thoughts?

Comment (davidbgk) [reply addok_france/utils.py:128]: Agreed, maybe an inline comment on top of the regexp?

Comment (yohanboniface) [reply addok_france/utils.py:128]: Done. :)

Comment (davidbgk) [reply addok_france/utils.py:128]: 👍

**Pull request merged**